src/H5ESint.c

```diff
@@ -50,6 +50,11 @@
 /* Local Typedefs */
 /******************/
 
+/* Callback context for get time estimate (gte) operations */
+typedef struct H5ES_gte_ctx_t {
+    uint64_t *time_est; /* Pointer to accumulated time estimate */
```

Comment:
It would be good to mention units both here and in the timeout field of H5ES_wait_ctx_t below.

Reply:
Done